Fix examples and upgrade FakerJS

pradeepnschrodinger opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
This fixes the site examples.
I'm also upgrading FakeJS package.

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
The examples in our site some fixes.
1. Image cells were blank cause the source URL provided by faker doesn't work without an upgrade.
2. The inputs in Autoscroll example throw errors on the console when the text is cleared.
3. The multiple tables in Fixed Rows Example goes out of sync if the user scrolls on any of the top 3 tables.
4. The footer example looks a bit weird without body rows filling the table.
5. Long Click example is supposed to highlight an entire row. But it only highlights an incomplete portion of every cell.
6. Max Height example resets the data when a new row is added.
7. Custom Styling example doesn't style body cells because it uses an non-existent style entry.

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
Tested local examples and made sure everything works fine.

## Screenshots (if appropriate):
<img width="1085" alt="image" src="https://github.com/schrodinger/fixed-data-table-2/assets/41563608/250c9b85-c397-4e91-92a9-3c4554fa8c8e">

<img width="1144" alt="image" src="https://github.com/schrodinger/fixed-data-table-2/assets/41563608/c4b58cb1-40ce-4723-b3d6-f90454f69c6e">

<img width="807" alt="image" src="https://github.com/schrodinger/fixed-data-table-2/assets/41563608/70d68484-5cb5-4b3a-9675-48eca8288447">

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

karry08 commented on examples/OwnerHeightExample.js:
```diff
@@ -17,7 +17,7 @@ class OwnerExample extends React.Component {
     this.state = {
       dataList: new FakeObjectDataListStore(100),
-      scrollY: 0,
+      pageYOffset: 0,
```
what was the issue in this example

pradeepnschrodinger replied:
`this.state.scrollY` doesn't exist. I'm pretty sure it was a typo and that it was meant to be `this.state.pageYOffset` instead.

Once we switch to typescript (atleast for the examples), we can avoid these situations.

karry08 commented on examples/MaxHeightExample.js:
```diff
@@ -13,23 +13,29 @@ class MaxHeightExample extends React.Component {
-    });
-  }
+  addRow = () => {
+    this.state.dataList.setSize(this.state.dataList.getSize() + 1);
```
does this also create a new data object for the newly added row?

karry08 replied:
nvm just saw changes in fakeobjectdataliststore file

pradeepnschrodinger replied:
Yea the actual row entries are only created on demand.

karry08 commented on src/FixedDataTableColumn.js:
```diff
@@ -168,6 +168,11 @@ class FixedDataTableColumn extends React.Component {
+    /**
+     *
+     */
+    cellClassName: PropTypes.string,
```
could you add comment why is this added

pradeepnschrodinger replied:
Good catch.

karry08 commented:
in the custom styling example there is scrollbarspacer in extreme right of the header we might might want to hide that

karry08 commented:
LGTM

pradeepnschrodinger commented:
> in the custom styling example there is scrollbarspacer in extreme right of the header we might might want to hide that
Fixed! I've updated the example (and screenshot).

pradeepnschrodinger commented on README.md:
```diff
@@ -1,4 +1,4 @@
-[Fixed Data Table 2](https://schrodinger.github.io/fixed-data-table-2/) for React &middot; ![Travic CI status](https://api.travis-ci.org/schrodinger/fixed-data-table-2.svg?branch=master) [![npm version](https://badge.fury.io/js/fixed-data-table-2.svg)](https://www.npmjs.com/package/fixed-data-table-2)
+[Fixed Data Table 2](https://schrodinger.github.io/fixed-data-table-2/) for React &middot; ![CI status](https://github.com/schrodinger/fixed-data-table-2/actions/workflows/build-and-run-tests.yml/badge.svg) &middot; [![npm version](https://badge.fury.io/js/fixed-data-table-2.svg)](https://www.npmjs.com/package/fixed-data-table-2)
```
This fixes the CI badge by having it generated from our `build-and-run-tests` workflow run. 

For context, we used travis long time back for CI, but we have replaced it with Github actions.
